### ISSUE TITLE
What Happens when Outputs' "Use Defaults" is clicked

### ISSUE THREAD

**Issue by schaluva** [DataBiosphere/saturn-ui#1158]
1. When you are at the Output tab of Tools, you can choose to "Use Defaults" for the Attribute column. If there are already custom attributes set, then clicking the "Use Defaults" does not have any action. You have to delete the fields, then click "Use Defaults" and the Attributes column is filled in with the same as the "Variable" value. This current functionality makes it a bit confusing as to what the "Use Defaults" button does. 
<img width="999" alt="screen shot 2019-02-06 at 4 34 20 pm" src="https://user-images.githubusercontent.com/43828038/52375424-9ef10000-2a2d-11e9-8c12-e8f512b280af.png">

2. It looks like when the WDL comes from Dockstore and not from the Method Repository, the "Use Defaults" option doesn't do anything, even when the fields are empty.

**Comment by bradtaylor** [comment]
Agreed. 'Use defaults' should overwrite the current value. That makes it clear what the default is. The user can always cancel.

Unfortunately, I don't see us getting to this soon, given all of our other critical work.

**Comment by schaluva** [comment]
@bradtaylor No problem, I'll let the user know that we have projected this for the future and follow-up with them when it gets implemented.
